Extensions/DraggableBehavior/tests/draggableruntimebehavior.spec.js

```diff
@@ -51,112 +51,211 @@ describe('gdjs.DraggableRuntimeBehavior', function () {
-      .onMouseButtonPressed(gdjs.InputManager.MOUSE_LEFT_BUTTON);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onMouseMove(850, 700);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame
-      .getInputManager()
-      .onMouseButtonReleased(gdjs.InputManager.MOUSE_LEFT_BUTTON);
-    runtimeScene.renderAndStep(1000 / 60);
-
-    expect(object.getX()).to.be(850);
-    expect(object.getY()).to.be(700);
-  });
-  it('should handle touches', function () {
-    runtimeGame.getInputManager().touchSimulateMouse(false);
-    object.setPosition(450, 500);
-
-    //Drag'n'drop
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onTouchStart(1, 10, 20);
-    runtimeGame.getInputManager().onTouchStart(0, 450, 500);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onFrameEnded();
-    runtimeGame.getInputManager().onTouchMove(0, 750, 600);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onFrameEnded();
-    runtimeGame.getInputManager().onTouchEnd(0);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onFrameEnded();
-
-    expect(object.getX()).to.be(750);
-    expect(object.getY()).to.be(600);
-
-    //Move another unrelated touch
-    runtimeGame.getInputManager().onTouchMove(1, 750, 600);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onTouchMove(1, 850, 700);
-    runtimeScene.renderAndStep(1000 / 60);
-
-    expect(object.getX()).to.be(750);
-    expect(object.getY()).to.be(600);
-
-    //Start drag'n'drop with another touch
-    runtimeGame.getInputManager().onTouchEnd(1);
-    runtimeGame.getInputManager().onFrameEnded();
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onTouchStart(1, 750, 600);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onTouchMove(1, 850, 700);
-    runtimeScene.renderAndStep(1000 / 60);
-    runtimeGame.getInputManager().onTouchEnd(1);
-    runtimeGame.getInputManager().onFrameEnded();
-
-    expect(object.getX()).to.be(850);
-    expect(object.getY()).to.be(700);
+  describe.skip('(mouse)', function () {
+    it('can drag an object', function () {
+      object.setPosition(450, 500);
+
+      //Drag'n'drop
+      runtimeScene.renderAndStep(1000 / 60);
+      runtimeGame.getInputManager().onMouseMove(450, 500);
+      runtimeGame
+        .getInputManager()
+        .onMouseButtonPressed(gdjs.InputManager.MOUSE_LEFT_BUTTON);
+      runtimeScene.renderAndStep(1000 / 60);
+      runtimeGame.getInputManager().onMouseMove(750, 600);
+      runtimeScene.renderAndStep(1000 / 60);
+      runtimeGame
+        .getInputManager()
+        .onMouseButtonReleased(gdjs.InputManager.MOUSE_LEFT_BUTTON);
+      runtimeScene.renderAndStep(1000 / 60);
+
+      expect(object.getX()).to.be(750);
+      expect(object.getY()).to.be(600);
+
+      //Mouse move without dragging
```

Comment:
nitpicking: convention for comments is a space after // and a period at the end (C# convention, though I think it's recommended in most languages: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#commenting-conventions)